docs/src/apireference.md

```diff
@@ -3,6 +3,10 @@ CurrentModule = MathOptInterface
 DocTestSetup = quote
     using MathOptInterface
     const MOI = MathOptInterface
+
+    # For compatibility with both Julia 1.0.5 and 1.5.2
```

Comment:
This is needed for Julia v1.0 but not for Julia v1 right ?

Reply:
Sorry, yes I mean v1.5 by v1. Having it work or both v1.0 and v1.5 is ideal of course but it would be nice to leave a comment here detailing whether this is needed for v1.0 or v1.5 so that we know what we do once v1.6 is the new LTS and we drop v1.0

Reply:
Added more comments as discussed